libs/agno/agno/vectordb/pineconedb/pineconedb.py

```diff
@@ -1,11 +1,27 @@
 from typing import Any, Dict, List, Optional, Union
 
 try:
+    import pinecone
```

Comment:
Rather only import `__version_` from pinecone and do the check first before the other pinecone imports?